sklearn/tests/test_common.py

```diff
@@ -270,3 +271,32 @@ def test_strict_mode_check_estimator():
 def test_strict_mode_parametrize_with_checks(estimator, check):
     # Ideally we should assert that the strict checks are Xfailed...
     check(estimator)
+
+
+# TODO: As more modules support get_feature_names_out they should be removed
+# from this list to be tested
+GET_FEATURES_OUT_MODULES_TO_IGNORE = [
```

Comment:
This is used so we can add the rest of the transformers in other PRs so we can keep this PR smaller.